components/safety_board/HW/HW_msp.c

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file HW_msp.c
+ * @brief  
+ * @author Joshua Lafleur (josh.lafleur@outlook.com)
+ * @version 0.1
+ * @date 2023-03-08
+ */
+
+
+/******************************************************************************
+ *                             I N C L U D E S
+ ******************************************************************************/
+
+#include "stm32f1xx.h"
+#include "SystemConfig.h"
+
+
+/******************************************************************************
+ *                       P U B L I C  F U N C T I O N S
+ ******************************************************************************/
+
+/**
+ * @brief  Called after HAL Init
+ */
+void HAL_MspInit(void)
+{
+    __HAL_RCC_AFIO_CLK_ENABLE();
+    __HAL_RCC_PWR_CLK_ENABLE();
+    HAL_NVIC_SetPriority(PendSV_IRQn, 15, 0);
```

Comment:
this should probably be higher priority? I believe a higher number here means lower priority